lib/packages.cf

```diff
@@ -94,6 +94,28 @@ body package_module yum
     #default_options => {};
 }
 
+body package_module freebsd_ports
+# @brief Define details used when interfacing with the freebsd ports package
+# module.
+#
+# **Note:** Ports are expected to be setup prior to trying to use the packages
+# promise. You may need to ensure that `portsnap extract` has been run.
+#
+# **Example:**
+# ```cf3
+# bundle agent main
+# {
+#   packages:
+#     freebsd::
```

Comment:
On that note, perhaps `freebsd_ports` should be renamed to simply `ports`. It's more in line with how the others are named and doesn't imply any particular platform. Unless you see it as very unlikely it will ever be used anywhere else.